modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/paths/CP2Utils.path

```diff
@@ -85,6 +85,11 @@
 	<td>//*[contains(@class, 'modal-content')]//*[text()='${key_text}']</td>
 	<td></td>
 </tr>
+<tr>
+	<td>TEXT</td>
+	<td>//*[contains(text(), '${key_text}')]</td>
+	<td></td>
+</tr>
```

Comment:
Hi @luanakarla would you please be more descriptive on this path? TEXT is too wide.

Reply:
Yea, done!